controller/src/functions/audio-renderings-bus.ts

```diff
@@ -83,6 +84,21 @@ export async function audioRenderedContentListener(
           context,
           row: parsed,
         });
+      } else {
+        // port / wacs non currently tracking audio info, but we still want to upsert if something changes from batch to batch since this is currently only route for content audio and renderings
```

Comment:
this is for audio but there's a similar branch of code for text renderings. Since Reuben said that PORT doesn't know these titles, this is a branch for updating anythign where PORT isn't the source of since the content table can be touched from both the rendering pipeline and from PORT direclty.